[17.0] [MIG] report_py3o: Migration to 17.0

bug2point0 opened the pull request.

antespi committed:
report_qweb_signer addon

pedrobaeza committed:
report_qweb_signer: Add feature request

OCA Transbot updated translations from Transifex

naousse committed:
mgmtsystem_kpi rename to kpi (#543)

* kpi migration to odoo 9
* UI error connected
* Correction base on last maxime review
* correction on @elicoidal review
* Warning exception improve in kpi_threshold
* Latest Ellicoidal comment implemented
* last Ellicoial recommendation done.
* Copyright corrected

hbrunn committed:
report_qweb_pdf_watermark (#86)

oca-transbot committed:
OCA Transbot updated translations from Transifex

StefanRijnhart committed:
[MIG] report_qweb_pdf_watermark

StefanRijnhart committed:
[ADD] Prioritize background transparency

oca-transbot committed:
OCA Transbot updated translations from Transifex

hbrunn committed:
[IMP] use PyPDF2 if possible

etobella committed:
[FIX] Fix Watermark from PNG

pedrobaeza committed:
[MIG] report_qweb_signer: Migration to 10.0

OCA Transbot updated translations from Transifex

oca-transbot committed:
OCA Transbot updated translations from Transifex

florian-dacosta committed:
Add module sql_export_mail

hbrunn committed:
[UPD] build environment

JordiBForgeFlow committed:
fix test_report_qweb_signer - tests were causing travis to stall

JordiBForgeFlow committed:
fix test_report_qweb_signer - tests were causing travis to stall

yajo committed:
[FIX] report_qweb_signer: Remove hardcoded memory limits

These limits were being hit when printing PDF reports with just 80 pages.

OCA Transbot updated translations from Transifex

[UPD] Update report_qweb_signer.pot

florian-dacosta committed:
Remove useless files

florian-dacosta committed:
styles and decorator fixes

ivantodorovich committed:
Increase version

ivantodorovich committed:
Highlight invalid ranges

ivantodorovich committed:
Added myself to contributors in README

ivantodorovich committed:
Module upgrade is enough

ivantodorovich committed:
Add tests

StefanRijnhart committed:
[FIX] Don't break printing some upstream reports with invalid models
When no background configuration is present on the report definition,
don't try to browse the report model especially because this breaks
some upstream reports (see https://github.com/odoo/odoo/pull/23389)

oca-transbot committed:
OCA Transbot updated translations from Transifex

oca-travis committed:
[UPD] Update report_qweb_pdf_watermark.pot

pedrobaeza committed:
[FIX] README syntax

schout-it committed:
[MIG] report_qweb_pdf_watermark: Migration to 11.0

AdriaGForgeFlow committed:
[MIG] kpi to v11

chienandalu committed:
[MIG] report_qweb_designer: Migration to 12.0

etobella committed:
[IMP] kpi: Add Kanban Views. Improve python code, Allow to access Odoo

JordiBForgeFlow committed:
[kpi] allow to return kpi and color in a single call without thresholds.

JordiBForgeFlow committed:
add periodicity in minutes

JordiBForgeFlow committed:
create kpi history as sudo

AdriaGForgeFlow committed:
[12.0][MIG] Migrate module kpi to v12

etobella committed:
[ADD] report_csv

JordiBForgeFlow committed:
[IMP] report_csv: use dialects

JordiBForgeFlow committed:
[IMP] report_csv: allow to evaluate print_report_name for multiple records

LoisRForgeFlow committed:
[IMP] report_csv: pass multi variable to be able to distingish which case you are in

[UPD] Update report_csv.pot

oca-travis committed:
[UPD] Update report_qweb_pdf_watermark.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-11.0/reporting-engine-11.0-report_qweb_pdf_watermark
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-11-0/reporting-engine-11-0-report_qweb_pdf_watermark/

pedrobaeza committed:
[MIG] report_qweb_signer: Refine migration

- Good dependency chain
- Context for forcing rendering PDF
- Extra test

[UPD] README.rst

[UPD] Update report_qweb_signer.pot

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-12.0/reporting-engine-12.0-report_qweb_signer
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-12-0/reporting-engine-12-0-report_qweb_signer/

[UPD] README.rst

florian-dacosta committed:
Migrate sql_export_mail to v9

florian-dacosta committed:
Migrate to v12

[UPD] Update sql_export_mail.pot

[UPD] README.rst

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: server-tools-12.0/server-tools-12.0-sql_export_mail
Translate-URL: https://translation.odoo-community.org/projects/server-tools-12-0/server-tools-12-0-sql_export_mail/

[ADD] icon.png

florian-dacosta committed:
Add group on cron field to avoid access error when user does not have the right to read crons.

Commit:
[12.0][MIG] report_csv

[UPD] Update report_csv.pot

[UPD] README.rst

florian-dacosta committed:
Add sql_export_excel

florian-dacosta committed:
Adapt module sql_export_mail to be compatible with multiple file format

florian-dacosta committed:
Migrate sql_export_excel to v12

OCA-git-bot committed:
[ADD] icon.png

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (68 of 68 strings)

Translation: reporting-engine-12.0/reporting-engine-12.0-kpi
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-12-0/reporting-engine-12-0-kpi/pt_BR/

florian-dacosta committed:
Small fixes for v12/python3 standard

Co-Authored-By: David Beal <david.beal@akretion.com>

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-12.0/reporting-engine-12.0-kpi
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-12-0/reporting-engine-12-0-kpi/

robin5210 committed:
[MIG] report_qweb_pdf_watermark: Migration to 12.0

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (79 of 79 strings)

Translation: reporting-engine-12.0/reporting-engine-12.0-kpi
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-12-0/reporting-engine-12-0-kpi/pt_BR/

kittiu committed:
report_async

oca-travis committed:
[UPD] Update report_qweb_pdf_watermark.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-12.0/reporting-engine-12.0-report_qweb_pdf_watermark
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-12-0/reporting-engine-12-0-report_qweb_pdf_watermark/

kongrattapong committed:
[IMP] report_csv: black, isort

kongrattapong committed:
[MIG] report_csv: Migration to 13.0

[UPD] Update report_csv.pot

[UPD] README.rst

report_csv 13.0.1.0.1

pre-commit update

lmignon committed:
[FIX] report_py3o: prettier xml after merge with master

lmignon committed:
[FIX] eslint error

sbidoul committed:
[FIX] make prettier happy

oca-travis committed:
Update report_async.pot

OCA-git-bot committed:
README.rst

OCA-git-bot committed:
report_async 12.0.1.0.1

Commit:
[IMP] kpi: black, isort

Commit:
Migration to 13.0

chienandalu committed:
[FIX] report_qweb_signer: reprint report crash

`render_qweb_pdf` must return a tuple of `(content, 'pdf')`

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-12.0/reporting-engine-12.0-report_qweb_pdf_watermark
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-12-0/reporting-engine-12-0-report_qweb_pdf_watermark/

oca-travis committed:
[UPD] Update sql_export_excel.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] icon.png

mstuttgart committed:
[FIX] Fix 'installable' syntax in manifest file

OCA-git-bot committed:
report_qweb_pdf_watermark 12.0.1.0.1

tupaq committed:
[IMP] report_csv: black, isort, prettier

tupaq committed:
[MIG] report_csv: Migration to 14.0

marcelsavegnago committed:
[FIX] report_qweb_signer

Fix website on manifest

fshah-initos committed:
[13.0][MIG] report_qweb_pdf_watermark

fshah-initos committed:
[IMP] report_qweb_pdf_watermark: black, isort, prettier

fshah-initos committed:
[14.0][MIG] report_qweb_pdf_watermark

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (51 of 51 strings)

Translation: reporting-engine-12.0/reporting-engine-12.0-report_async
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-12-0/reporting-engine-12-0-report_async/pt/

legalsylvain committed:
[FIX] sql_export_excel: add missing dependency to python lib openpyxl

oca-travis committed:
[UPD] Update report_qweb_pdf_watermark.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
sql_export_excel 12.0.1.1.0

oca-travis committed:
[UPD] Update kpi.pot

OCA-git-bot committed:
[UPD] README.rst

hkapatel-initos committed:
[IMP] sql_export_mail: black, isort, prettier

hkapatel-initos committed:
[MIG] sql_export_mail: Migration to 14.0

[UPD] Update sql_export_mail.pot

[UPD] README.rst

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: server-tools-14.0/server-tools-14.0-sql_export_mail
Translate-URL: https://translation.odoo-community.org/projects/server-tools-14-0/server-tools-14-0-sql_export_mail/

[UPD] Update sql_export_mail.pot

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: server-tools-14.0/server-tools-14.0-sql_export_mail
Translate-URL: https://translation.odoo-community.org/projects/server-tools-14-0/server-tools-14-0-sql_export_mail/

[UPD] Update sql_export_mail.pot

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: server-tools-14.0/server-tools-14.0-sql_export_mail
Translate-URL: https://translation.odoo-community.org/projects/server-tools-14-0/server-tools-14-0-sql_export_mail/

hkapatel-initos committed:
[IMP] sql_export_excel: black, isort, prettier

hkapatel-initos committed:
[MIG] sql_export_excel: Migration to 14.0

Commit:
[IMP] report_qweb_signer: Black python code

Commit:
[MIG] report_qweb_signer: Migration to 13.0

[UPD] Update report_qweb_signer.pot

[UPD] README.rst

Commit:
[IMP] report_qweb_signer add endesive

sbidoul committed:
Apply dotfiles

sbidoul committed:
Apply dotfiles

OCA-git-bot committed:
kpi 13.0.1.0.1

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update sql_export_excel.pot

ThomasBinsfeld committed:
[IMP] report_qweb_signer: allowed reports on certificate
Add a new 'Allowed reports' field on the certificate. Only reports listed in this field can be signed. No report means all reports are allowed.

omar7r committed:
[MIG] report_qweb_signer: Migration to 14.0 updating java's lib for visible signatures

oca-travis committed:
[UPD] Update sql_export_excel.pot

chienandalu committed:
[FIX] report_qweb_signer: noupdate defaults

We don't want our custom parameters be reset every time the module
updates.

TT32256

omar7r committed:
[IMP] report_qweb_signer: Signature visible parameter not include by default and fix multicompany issue

omar7r committed:
[FIX] report_qweb_signer: Fix when report is printed without related objects (print from wizard in some cases)

newtratip committed:
report_async: black, isort, prettier

newtratip committed:
report_async

yankinmax committed:
[IMP] sql_export_excel: pylint

yankinmax committed:
[MIG] sql_export_excel: Migration to 15.0

Commit:
[UPD] Update sql_export_excel.pot

OCA-git-bot committed:
[UPD] README.rst

ofonseca-pyming committed:
Added translation using Weblate (Catalan)

omar7r committed:
[FIX] report_qweb_signer: Handle multi-company correctly

Commit:
[IMP] report_csv : black, isort, prettier

Commit:
[MIG] report_csv: Migration to 15.0

Commit:
Update report_async.pot

OCA-git-bot committed:
README.rst

etobella committed:
[IMP] report_qweb_signer: keep env company if company is not defined

etobella committed:
[IMP] report_qweb_signer: Add default-jre-headless for GH

fernandahf committed:
[REF] report_qweb_pdf_watermark: remove PyPDF2 from
external_dependencies

OCA-git-bot committed:
report_qweb_pdf_watermark 14.0.1.0.1

Hardik-OSI committed:
report_async: Pre-Commit

Hardik-OSI committed:
report_async: migration to 15.0

jabelchi committed:
Added translation using Weblate (Catalan)

jabelchi committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (17 of 17 strings)

Translation: reporting-engine-15.0/reporting-engine-15.0-sql_export_excel
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-15-0/reporting-engine-15-0-sql_export_excel/ca/

ThomasBinsfeld committed:
[ADD] report_qweb_signer: add  pedrobaeza idea in ROADMAP

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-14.0/reporting-engine-14.0-report_qweb_signer
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-14-0/reporting-engine-14-0-report_qweb_signer/

legalsylvain committed:
[MIG] sql_export_mail from OCA/server-tools 14.0 to OCA/reporting-engine 16.0 branch
- Update views, regarding changes in sql_export module
- Add prepare function for cron vals

[UPD] Update sql_export_mail.pot

[UPD] README.rst

igallart committed:
[14.0][FIX] report_qweb_signer: Fix TypeError

TypeError: 'int' object is not subscriptable

bosd committed:
[MIG] report_qweb_pdf_watermark: Migration to 15.0

[MIG] report_qweb_pdf_watermark: Migration to 15.0

AnizR committed:
[MIG] report_csv: Migration to 16.0

[UPD] Update report_csv.pot

[UPD] README.rst

florian-dacosta committed:
[16.0][MIG] sql_export_excel : migration to v16

Commit:
[UPD] Update report_qweb_pdf_watermark.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-15.0/reporting-engine-15.0-report_qweb_pdf_watermark
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-15-0/reporting-engine-15-0-report_qweb_pdf_watermark/

AungKoKoLin1997 committed:
[IMP] report_csv: add encoding option

[UPD] Update report_csv.pot

[BOT] post-merge updates

florian-dacosta committed:
[IMP] Add back the variable feature in sql_export
Refactore to use the new properties field

florian-dacosta committed:
[FIX] email template on sql_export_mail

florian-dacosta committed:
[FIX] Manage jsonb fields

bvanjeelharia committed:
[MIG] report_qweb_pdf_watermark: Migration to 16.0

Syntax Fixes

Commit:
Update report_async.pot

OCA-git-bot committed:
README.rst

Nikul-OSI committed:
[MIG] report_async: Migrated v15 to v16

mymage committed:
Translated using Weblate (Italian)

Currently translated at 26.0% (12 of 46 strings)

Translation: reporting-engine-14.0/reporting-engine-14.0-report_qweb_signer
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-14-0/reporting-engine-14-0-report_qweb_signer/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 29.1% (23 of 79 strings)

Translation: reporting-engine-13.0/reporting-engine-13.0-kpi
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-13-0/reporting-engine-13-0-kpi/it/

Commit:
[UPD] Update report_qweb_pdf_watermark.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-16.0/reporting-engine-16.0-report_qweb_pdf_watermark
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-16-0/reporting-engine-16-0-report_qweb_pdf_watermark/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 29.1% (23 of 79 strings)

Translation: reporting-engine-13.0/reporting-engine-13.0-kpi
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-13-0/reporting-engine-13-0-kpi/it/

sysadminmatmoz committed:
Translated using Weblate (Slovenian)

Currently translated at 63.0% (29 of 46 strings)

Translation: reporting-engine-14.0/reporting-engine-14.0-report_qweb_signer
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-14-0/reporting-engine-14-0-report_qweb_signer/sl/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 29.1% (23 of 79 strings)

Translation: reporting-engine-13.0/reporting-engine-13.0-kpi
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-13-0/reporting-engine-13-0-kpi/it/

Commit:
Update CONFIGURE.rst

florian-dacosta committed:
[IMP] sql_export_mail : Add migration script to fix email template

[UPD] Update sql_export_mail.pot

sql_export_mail 16.0.1.0.1

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-16.0/reporting-engine-16.0-sql_export_mail
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-16-0/reporting-engine-16-0-sql_export_mail/

legalsylvain committed:
[FIX] sql_export_mail : do not raise an error if non admin user want to access to sql export

[UPD] README.rst

[BOT] post-merge updates

[BOT] post-merge updates

rousseldenis committed:
[FIX] report_csv: Assign 'reportname' variable before accessing it

rousseldenis committed:
[FIX] report_csv: Return werkzeug Exception

rousseldenis committed:
[IMP] report_csv: Add controller tests

report_csv 16.0.1.0.1

[UPD] README.rst

Commit:
[UPD] Update report_async.pot

OCA-git-bot committed:
[UPD] README.rst

jabelchi committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (46 of 46 strings)

Translation: reporting-engine-14.0/reporting-engine-14.0-report_qweb_signer
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-14-0/reporting-engine-14-0-report_qweb_signer/ca/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (10 of 10 strings)

Translation: reporting-engine-16.0/reporting-engine-16.0-report_qweb_pdf_watermark
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-16-0/reporting-engine-16-0-report_qweb_pdf_watermark/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (53 of 53 strings)

Translation: reporting-engine-16.0/reporting-engine-16.0-report_async
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-16-0/reporting-engine-16-0-report_async/es/

BT-rmartin committed:
[IMP] Avoiding ERROR in log with tests of report_qweb_pdf_watermark

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

Commit:
[UPD] Update sql_export_excel.pot

OCA-git-bot committed:
[BOT] post-merge updates

sbejaoui committed:
[16.0][IMP] report_csv: support attachment use

[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-13.0/reporting-engine-13.0-kpi
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-13-0/reporting-engine-13-0-kpi/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-14.0/reporting-engine-14.0-report_qweb_signer
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-14-0/reporting-engine-14-0-report_qweb_signer/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-16.0/reporting-engine-16.0-report_async
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-16-0/reporting-engine-16-0-report_async/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (17 of 17 strings)

Translation: reporting-engine-16.0/reporting-engine-16.0-sql_export_excel
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-16-0/reporting-engine-16-0-sql_export_excel/es/

HviorForgeFlow committed:
[IMP] pre-commit lint

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-14.0/reporting-engine-14.0-report_qweb_signer
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-14-0/reporting-engine-14-0-report_qweb_signer/

mymage committed:
Added translation using Weblate (Italian)

benwillig committed:
[MIG] report_qweb_signer to 16.0

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

Translated using Weblate (Italian)

Currently translated at 100.0% (14 of 14 strings)

Translation: reporting-engine-16.0/reporting-engine-16.0-report_csv
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-16-0/reporting-engine-16-0-report_csv/it/

legalsylvain committed:
[IMP] bi_sql_editor: allow to generate form view. So It is possible to see data detail, when clicking on an item in a tree view
- update / create tree and form view once all the models are loaded, to avoid errors because fields are not available

legalsylvain committed:
[IMP] bi_sql_editor : Allow to use sum / avg operator on tree view

sanderlienaerts committed:
[IMP] report_qweb_pdf_watermark: black, isort, prettier

sanderlienaerts committed:
[MIG] report_qweb_pdf_watermark: Migration to 17.0

sanderlienaerts committed:
[IMP] sql_export_mail: black, isort, prettier

sanderlienaerts committed:
[MIG] sql_export_mail: Migration to 17.0

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (17 of 17 strings)

Translation: reporting-engine-16.0/reporting-engine-16.0-sql_export_excel
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-16-0/reporting-engine-16-0-sql_export_excel/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (53 of 53 strings)

Translation: reporting-engine-16.0/reporting-engine-16.0-report_async
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-16-0/reporting-engine-16-0-report_async/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 28.9% (22 of 76 strings)

Translation: reporting-engine-13.0/reporting-engine-13.0-kpi
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-13-0/reporting-engine-13-0-kpi/it/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-16.0/reporting-engine-16.0-report_qweb_signer
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-16-0/reporting-engine-16-0-report_qweb_signer/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (46 of 46 strings)

Translation: reporting-engine-16.0/reporting-engine-16.0-report_qweb_signer
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-16-0/reporting-engine-16-0-report_qweb_signer/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Translated using Weblate (Spanish)

Currently translated at 100.0% (14 of 14 strings)

Translation: reporting-engine-16.0/reporting-engine-16.0-report_csv
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-16-0/reporting-engine-16-0-report_csv/es/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (46 of 46 strings)

Translation: reporting-engine-16.0/reporting-engine-16.0-report_qweb_signer
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-16-0/reporting-engine-16-0-report_qweb_signer/it/

sakthi-shan committed:
[IMP] report_async: pre-commit auto fixes

sakthi-shan committed:
[MIG] report_async: Migration to 17.0

glitchov committed:
[IMP] add missing _is_report_type_signable to allow signing other types of pdf

bug2point0 committed:
[IMP] replace deprecated pkg_resources with importlib.resources

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (62 of 62 strings)

Translation: reporting-engine-17.0/reporting-engine-17.0-base_comment_template
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-17-0/reporting-engine-17-0-base_comment_template/es_AR/

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (64 of 64 strings)

Translation: reporting-engine-17.0/reporting-engine-17.0-sql_export
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-17-0/reporting-engine-17-0-sql_export/es_AR/

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (43 of 43 strings)

Translation: reporting-engine-17.0/reporting-engine-17.0-sql_request_abstract
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-17-0/reporting-engine-17-0-sql_request_abstract/es_AR/

angelmoya reviewed (approved):
:+1:  LGTM tested on runbot ok

Commit:
[UPD] Update report_async.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-17.0/reporting-engine-17.0-report_async
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-17-0/reporting-engine-17-0-report_async/

victoralmau committed:
[IMP] report_qweb_parameter: Remove warning Found deprecated directive @t-raw

WARNING devel odoo.addons.base.models.ir_qweb: Found deprecated directive
@t-raw='docs.env["ir.qweb"].check_length(docs[0].vat, 10, False)[:10]' in template 237.
Replace by @t-out, and explicitely wrap content in `Markup` if necessary
(which likely is not the case)

WARNING devel odoo.addons.base.models.ir_qweb: Found deprecated directive
@t-raw='docs.env["ir.qweb"].check_length(docs[0].company_registry, False, 10)'
in template 237. Replace by @t-out, and explicitely wrap content in `Markup`
if necessary (which likely is not the case)

victoralmau committed:
[IMP] report_qweb_parameter: pre-commit

Execute pre-commit to remove the existing warning.

WARNING prod py.warnings: /opt/odoo/custom/src/odoo/odoo/addons/base/models/ir_module.py:178:
DeprecationWarning: XML declarations in HTML module descriptions are deprecated since Odoo 17,
report_qweb_parameter can just have a UTF8 description with not need for a declaration.

OCA-git-bot committed:
[BOT] post-merge updates

EdgarRetes committed:
[IMP] kpi: pre-commit auto fixes

Tardo commented on report_py3o/__manifest__.py:
```diff
@@ -0,0 +1,29 @@
+    "depends": ["web"],
+    "external_dependencies": {
+        "python": ["py3o.template", "py3o.formats"],
+        "deb": ["libreoffice"],
```
More info: https://github.com/odoo/odoo/blob/d759264c6e9b7a8cc8b6cffdc5509df654cd434c/odoo/modules/module.py#L508

```suggestion
        "bin": ["libreoffice"],
```

I think this should be changed on all versions... and other modules...

bug2point0 replied:
[odoo docs](https://www.odoo.com/documentation/17.0/developer/reference/backend/module.html?highlight=external_dependencies) are pretty clear on this. Nice catch.

Tardo replied:
Umm... looks like OCA "unofficial" way (i don't know where is documented) to handle dependencies in CI env: https://github.com/OCA/oca-ci/blob/c10c90835d3baf6576faaae03f593099b9995009/bin/oca_install_addons#L16

Perhaps a solution could be to use ```bin``` to have the Odoo check and ```deb``` to indicate how to install the package with the 'apt package manager'?

ping @sbidoul

sbidoul replied:
You can add `bin` if you want, but please keep `deb` so libreoffice is installed in ci and runboat.

That said, I'm not sure libreoffice is a hard dependency. IIRC it's optional when using `report_py3o_fusion_server`, so in that case we should not add `bin`.

Tardo replied:
@bug2point0 I think there should be both “bin” and “deb”. 
As I see it, actually it all depends on which module you give more importance to 'report_py3o_fusion_server' vs 'report_py3o'... Everything else is a mess of dependencies (if you choose X with Y but you have Z then disable W... :/). At least from what I could see.

You decide, if you choose only “deb” it seems right to me... and if you choose “bin” and “deb” the same. Little more I can say.

bug2point0 committed:
binary dependency enforcement

Co-authored-by: Alexandre Díaz <dev@redneboa.es>

sersanchus commented:
I think I've found an issue. There is a demo report for res.users model that is defined in the file 'demo/res_users_report_py3o.xml'. However, it is not added in the 'demo' section of the '\_\_manifest\_\_.py' but rather in 'data'. Therefore, it is always installed even if demos are not required for the module.

Commit:
Added translation using Weblate (French)

EdgarRetes committed:
[MIG] kpi: Migration to 17.0

Commit:
[UPD] Update bi_sql_editor.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-17.0/reporting-engine-17.0-bi_sql_editor
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-17-0/reporting-engine-17-0-bi_sql_editor/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (144 of 144 strings)

Translation: reporting-engine-17.0/reporting-engine-17.0-bi_sql_editor
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-17-0/reporting-engine-17-0-bi_sql_editor/it/

Commit:
[UPD] Update report_qweb_pdf_watermark.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (10 of 10 strings)

Translation: reporting-engine-17.0/reporting-engine-17.0-report_qweb_pdf_watermark
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-17-0/reporting-engine-17-0-report_qweb_pdf_watermark/it/

Commit:
[UPD] Update kpi.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-17.0/reporting-engine-17.0-kpi
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-17-0/reporting-engine-17-0-kpi/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (74 of 74 strings)

Translation: reporting-engine-17.0/reporting-engine-17.0-kpi
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-17-0/reporting-engine-17-0-kpi/it/

mtbochaca committed:
Translated using Weblate (Catalan)

Currently translated at 63.8% (92 of 144 strings)

Translation: reporting-engine-17.0/reporting-engine-17.0-bi_sql_editor
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-17-0/reporting-engine-17-0-bi_sql_editor/ca/

Commit:
[UPD] Update sql_export_mail.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: reporting-engine-17.0/reporting-engine-17.0-sql_export_mail
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-17-0/reporting-engine-17-0-sql_export_mail/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (14 of 14 strings)

Translation: reporting-engine-17.0/reporting-engine-17.0-sql_export_mail
Translate-URL: https://translation.odoo-community.org/projects/reporting-engine-17-0/reporting-engine-17-0-sql_export_mail/it/

ValentinVinagre committed:
[FIX] report_qweb_parameter: out and raw doesn't work with conditionals

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
[IMP] update dotfiles

victoralmau committed:
[IMP] report_csv: pre-commit auto fixes

victoralmau committed:
[MIG] report_csv: Migration to 17.0

TT52352

mariusnick committed:
[MIG] sql_report_excel: migration to 17.0

Commit:
[UPD] Update report_csv.pot

OCA-git-bot committed:
[BOT] post-merge updates

victoralmau committed:
[DCK] report_csv: Add development_status key to manifest

OCA-git-bot committed:
[BOT] post-merge updates

len-foss committed:
[REF] report_qweb_signer: make signing functions reusable

carlos-lopez-tecnativa committed:
[IMP] report_qweb_signer: pre-commit auto fixes

carlos-lopez-tecnativa committed:
[MIG] report_qweb_signer: Migration to version 17.0

Commit:
[UPD] Update report_qweb_signer.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update sql_export_excel.pot

OCA-git-bot committed:
[BOT] post-merge updates

Tardo commented:
@bug2point0 please, can you rebase? thanks!

bug2point0 committed:
Add report_py3o

Tardo commented:
@bug2point0 please, can you rebase? thanks! :B ... and please, check the commits because it looks like you're touching a lot more files than the migration requires.

pilarvargas-tecnativa commented:
superseded by https://github.com/OCA/reporting-engine/pull/1021
